### PR TITLE
Technical => main

### PR DESCRIPTION
**Pull request by Mattstar64**
Final version of the Technical specification, also including naming conventions.

### CODE REVIEW

**Review by GuillaumeDespaux — CHANGES_REQUESTED** [review]
On ./technical/naming_conventions_and_review.md :
- minuscule is a French word
- Line 32 "exmple" (typo)

./technical/technical_specification.md :
- Missing date for Finished
- L19 to L24 Capital letter missing on some letters
- L48 "technologie's" (typo)
- L53 "movement" (typo)
- L58 "wining" (typo)
- L61 "high scores" (typo)
- L80 not the objective
- L103 "-" error
- L115 & L118 you don't apply your own rule
- L122 & L126 //
- L130 Ok only if script to lunch the game else in ./src/
- L131 typo
- L143 not English -> At least "In technical details"
- L149 "developpement" (typo)
- L165 Switches -> switches
- L166 show -> display
- L167 -> current level.
- L185 check for another way to say it (we render)
- L193 image? not pixel?
- L218 remove the "The" to put "Differents cars"
- L246 French commentary
- L279
- L281 & L283 use Up and Down
- L313 Movements
- L334 & 338 & 340 why dot at the end ?
- L349 begin -> start
- L351 bins -> the trash cans
- L391 & L393 high scores (typo)

**Review by Enonome — CHANGES_REQUESTED** [review]
- In the naming_conventions_and_rules, you wrote "Varibles" instead of "Variables" (line 19)
- In the 4fh dot of 2.2 in technical_spec, there are a useless dash before "Team Members"
- In your file structure, the README is written like that "READEME"
- You've forgotten the "Go" for the Lenovo Thinkpad between 512 and SSD
- You wrote (3.4, B) "o control" instead of "To control" at the first line
- In the description of the 4th code, you wrote "whre switches" instead of "where switches"
- You've forgotten a 'o' at "loose" (so 'lose') in the C sections

**Comment by Mattstar64** [comment]
fixed

File: documents/technical/technical_specification.md [hunk 1]
```diff
@@ -44,21 +44,21 @@
...
+ - [**3.4 Controls and characters movements**](#34-controls-and-movements)
    - [**A. Car movements**](#a-car-movements)
    - [**B. Raccoon controls**](#b-raccoon-controls)
  - [**3.5 Game background**](#35-game-background)
  - [**3.6 Game mechanics**](#36-game-mechanics)
    - [**A. Wining and loosing mechanics**](#a-winning-and-loosing-mechanics)
    - [**B. Starting and finishing zone**](#b-starting-and-finishing-zone)
    - [**C. Lives**](#c-lives)
-   - [**D. Points and high scores**](#d-points-and-high-scores)
+   - [**D. Points and high score**](#d-points-and-high-scores)
```
Comment (Enonome) [inline documents/technical/technical_specification.md]: Problem with redirection (not the right tag)

File: documents/technical/technical_specification.md [hunk 2]
```diff
@@ -17,11 +17,11 @@
...
 |1.0|09/27/2024|Mattéo LEFIN| - Create documents and finished Overview.|
-1.1| 10/01/2024|Mattéo LEFIN| - add Overview and conventions |
-1.2| 10/02/2024|Mattéo LEFIN| - Remade the table of content <br> - finished 3.1/2/3.B |
-1.3| 10/08/2024|Mattéo LEFIN| - added code in 3.5 |
-1.4| 10/10/2024|Mattéo LEFIN| - finished writing <br> - added winning/loosing schema <br> - added all code|
-1.5 | 10/11/2024|Mattéo LEFIN| - Text polishing <br> - add images <br> - delete glossary |
+1.1| 10/01/2024|Mattéo LEFIN| - Add Overview and Conventions |
+1.2| 10/02/2024|Mattéo LEFIN| - Remade the Table of content <br> - finished 3.1/2/3.B |
+1.3| 10/08/2024|Mattéo LEFIN| - Added code in 3.5 |
+1.4| 10/10/2024|Mattéo LEFIN| - Finished writing <br> - added winning/loosing schema <br> - Added all code|
```
Comment (Enonome) [inline documents/technical/technical_specification.md]: You can also put a capital letter on the 1st "added"

added -> Added

File: documents/technical/technical_specification.md [hunk 3]
```diff
@@ -17,11 +17,11 @@
 |Version|Date|Author|Description|
 |-------|----|---|--|
 |1.0|09/27/2024|Mattéo LEFIN| - Create documents and finished Overview.|
-1.1| 10/01/2024|Mattéo LEFIN| - add Overview and conventions |
-1.2| 10/02/2024|Mattéo LEFIN| - Remade the table of content <br> - finished 3.1/2/3.B |
-1.3| 10/08/2024|Mattéo LEFIN| - added code in 3.5 |
-1.4| 10/10/2024|Mattéo LEFIN| - finished writing <br> - added winning/loosing schema <br> - added all code|
-1.5 | 10/11/2024|Mattéo LEFIN| - Text polishing <br> - add images <br> - delete glossary |
+1.1| 10/01/2024|Mattéo LEFIN| - Add Overview and Conventions |
+1.2| 10/02/2024|Mattéo LEFIN| - Remade the Table of content <br> - finished 3.1/2/3.B |
```
Comment (Enonome) [inline documents/technical/technical_specification.md]: You can put a capital letter on "finished" too

finished -> Finished

File: documents/technical/technical_specification.md [hunk 4]
```diff
@@ -44,21 +44,21 @@
...
  - [**3.3 Drawing in VGA**](#33-drawing-in-vga)
    - [**A. Graphics**](#a-graphics)
    - [**B. Sprites**](#b-sprites)
- - [**3.4 Controls and characters movement**](#34-controls-and-movements)
+ - [**3.4 Controls and characters movements**](#34-controls-and-movements)
    - [**A. Car movements**](#a-car-movements)
    - [**B. Raccoon controls**](#b-raccoon-controls)
  - [**3.5 Game background**](#35-game-background)
  - [**3.6 Game mechanics**](#36-game-mechanics)
    - [**A. Wining and loosing mechanics**](#a-winning-and-loosing-mechanics)
```
Comment (Enonome) [inline documents/technical/technical_specification.md]: "Wining" takes two 'n'

wining -> winning

File: documents/technical/technical_specification.md [hunk 5]
```diff
@@ -140,60 +140,60 @@ Below is our file structure plan :
 
 <summary>3. Into technicalities</summary>
```
Comment (Enonome) [inline documents/technical/technical_specification.md]: Change the name of the summary in order to correspond with the current name of the 3.

Into technicalities -> in technical details

File: documents/technical/technical_specification.md [hunk 6]
```diff
@@ -140,60 +140,60 @@ Below is our file structure plan :
...
 
-### 3.1 Used Technologies.
+### 3.1 Used Technologies
 
-#### A. Used computers.
+#### A. Used computers
 
-For this project, we used for developpment :
+For this project, we used for developpement :
 - LENOVO ThinkPad 21JKCTO1WW - Core i7 - 16 Go RAM - 512 SSD
```
Comment (Enonome) [inline documents/technical/technical_specification.md:150]: Missing the "Go" for Giga octet between 512 and SSD

**Review by Enonome — CHANGES_REQUESTED** [review]
- Line 149, "developpement" is a French word, try to use "development" 
- Lines 151 & 152, the two lines use "Code ..." instead of "Core"
- Line 155, you can put a lower case in "Screen" **(Screen -> screen)**
- Line 163, you can use a comma after "On this board"
- Line 167, put "the" before "current score"
- Line 177 : 
> - You can joined "set" and "up" ("set up" -> "setup")
> - The "the" before the link is too much
> - You can use "the" before "Nandland website"
- Line 187, the 's' of "sync" in "V_Sync_Pulse" can be in capital letter
- Line 216 and 277, you can put a dot after the letters 
- Line 366, my bad, loose is the adjective or we need to use the verb

File: documents/technical/technical_specification.md (hunk 6 above)
Comment (Mattstar64) [reply documents/technical/technical_specification.md:150]: the Go isn't placed here for this one, it's actually normal

File: documents/technical/technical_specification.md (hunk 5 above)
Comment (Mattstar64) [reply documents/technical/technical_specification.md]: Already done ?

File: documents/technical/technical_specification.md (hunk 6 above)
Comment (Enonome) [reply documents/technical/technical_specification.md:150]: Okay 👍

File: documents/technical/technical_specification.md (hunk 1 above)
Comment (Enonome) [reply documents/technical/technical_specification.md]: Fixed

File: documents/technical/technical_specification.md (hunk 5 above)
Comment (Enonome) [reply documents/technical/technical_specification.md]: Yeah, probably a display issue when reviewing

File: documents/technical/technical_specification.md [hunk 7]
```diff
@@ -0,0 +1,460 @@
...
+
+<summary>3. In technical details</summary>
+
+## 3. In technical details
+
+### 3.1 Used Technologies
+
+#### A. Used computers
+
+For this project, we used for developpment :
```
Comment (Enonome) [inline documents/technical/technical_specification.md]: There is an extra 'p' in "developpment'

(developpment -> development)

**Review by Enonome — CHANGES_REQUESTED** [review]
My last overall review did not verify (note there is a new pending comment)

**Review by Enonome — APPROVED** [review]
Everything is good for me

**Review by GuillaumeDespaux — APPROVED** [review]
Ok for me.

**Pull request merged**